### DISCUSSION TITLE
DITA-OT rendering issues | Japanese text in an English PDF is shown as ####.

### DISCUSSION THREAD

**Discussion by DeekshaBhagat** [dita-ot/dita-ot#4491] [Q&A]
![LangIssueDITAOT](https://github.com/user-attachments/assets/76797d17-58cf-4118-880c-eb2896a776d7)
Hi All,

Use Case - 
We have a ditamap and a topic as added below.
The ditamap and topic are in English language. We added some japanese language characters.

1. Render it using dita-ot pdf2 plugin in Wnidows.
    Rendering is successful, with output showing maptitle as - Map1_更新しました
   
2. Render the same using dita-ot pdf2 plugin in Linux
    Rendering is successful, with output showing maptitle as - Map1_######

Observation - The windows supports te japanese or multilingual support, but linux doesnot.
Question - How can we add support for linux too?
I am looking for solution from last one week.

Data to recreate te issue()
--------------------------------------------------------------------------------------------------------------------------------------------------------
Map.ditamap
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
<map id="map">
    <topicmeta id="sdm:n5dg05b3w7">
        <navtitle id="sdm:3h7y2t8pskj">Map1_更新しました</navtitle>
    </topicmeta>
     <topicref id="sdm:63vspea5gjh" href="./Topic.dita"/>
</map>
--------------------------------------------------------------------------------------------------------------------------------------------------------
Topic.dita
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
<topic id="topic" >
    <title id="sdm:lp4gop9fj1b">段落_Topic1_更新しました</title>
    <body id="sdm:mrz09oaxhbh">
        <p id="sdm:b2rtm2oggca">desc段落_</p>
    </body>
</topic>
-------------------------------------------------------------------------------------------------------------------------------------------------------- 
DITA-OT version 4.2.2
--------------------------------------------------------------------------------------------------------------------------------------------------------
Command
dita --input=.\ParameterMap.ditamap --format=pdf2 --output=.\out --temp=.\logs --clean.temp=no -d
--------------------------------------------------------------------------------------------------------------------------------------------------------
Error - 
[WARN] Font "STIXTwoMath,normal,700" not found. Substituting with "STIXTwoMath,normal,400". 
[WARN] Glyph "更" (0x66f4) not available in font "Helvetica-Bold". 
[WARN] Glyph "新" (0x65b0) not available in font "Helvetica-Bold". 
[WARN] Glyph "し" (0x3057, sihiragana) not available in font "Helvetica-Bold". 
[WARN] Glyph "ま" (0x307e, mahiragana) not available in font "Helvetica-Bold". 
![image](https://github.com/user-attachments/assets/5e74cd92-e085-4d32-85ae-c72a63f4145a)

[WARN] Glyph "た" (0x305f, tahiragana) not available in font "Helvetica-Bold". 
[INFO] Rendered page #1.

**Comment by stefan-jung** [comment]
The error message is clear. Helvetica does not have the glyphs to render the Japanese text. Provide a Japanese font, e.g. by using the Adobe Source Han fonts, and set the font-family attributes in your custom PDF plugin.

**Reply by DeekshaBhagat** [reply]
But, same plugin works in windows  and gives errors in linux.
So, it this really a plugin issue or OS issue?

**Reply by DeekshaBhagat** [reply]
The issue is fixed.
I have downloade Google fonts "[Noto Sans Japanese](https://fonts.google.com/noto/specimen/Noto+Sans+JP)" and added to my plugin. It is working now.

:)
